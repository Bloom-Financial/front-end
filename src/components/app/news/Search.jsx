import React from 'react';
import PropTypes from 'prop-types';

const Search = ({ search, onChange }) => (
  <input
    // className={}
    type="text"
    placeholder="Search here..."
    value={search}
    onChange={onChange}
  />
);

Search.propTypes = {
  search: PropTypes.string.isRequired,
  onChange: PropTypes.func.isRequired,
};

export default Search;
