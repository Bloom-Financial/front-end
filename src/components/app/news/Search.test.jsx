import React from 'react';
import { render } from '@testing-library/react';
import Search from './Search';


describe('Test the search component', () => {
  it('tests the search component', () => {
    const { asFragment } = render(<Search search="help"/>);
    expect(asFragment()).toMatchSnapshot();
  });
});
