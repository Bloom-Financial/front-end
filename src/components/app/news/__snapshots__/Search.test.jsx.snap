// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test the search component tests the search component 1`] = `
<DocumentFragment>
  <input
    class="search"
    placeholder="Search here..."
    style="height: 2vh; width: 10vw;"
    type="text"
    value="help"
  />
</DocumentFragment>
`;
